build.gradle

```diff
@@ -1266,6 +1272,23 @@ project(':group-coordinator') {
   javadoc {
     enabled = false
   }
+
+  task processMessages(type:JavaExec) {
+    mainClass = "org.apache.kafka.message.MessageGenerator"
+    classpath = configurations.generator
+    args = [ "-p", "org.apache.kafka.coordinator.group.generated",
```

Comment:
I noticed that sometimes we append `.generated` in similar closures and sometimes we do not. Are we appending `.generated` going forward?

Reply:
Yeah, I have noticed this as well. Personally, I like having `generated` in the package name because it makes the intent clear in the code. I am also fine if folks prefer to remove it.